### PR TITLE
[SANW-19] ReadMe

### PR DESCRIPTION
**Pull request by timalsinab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,119 @@
...
+
+- **Objectives:**
+A website that serves as the front-facing information and involvement page for anyone wanting to know more about us and our goals. It will have many features such as a mission statement, timelines, ways to get involved, contact information, and donations.
+
+- **Love:** Building our foundation, we start everything with love. From peTeaching children to love themselves, and those whom they are surrounded with, will give them a lifetime of lessons- especially to help each other in need. Honoring a natural lifestyle, we will teach children to grow their own food, living in an organic lifestyle, which involves a language of love that is able to be passed down from generations to generations. Everything starts with a vision, and this one is an impeccable one; touching lives of many. We are starting from the ground up, and you can help us take this vision- make it a physical reality.
```
[the remark this answers is not in the document]
Comment (timalsinab) [reply README.md:31]: Will do it.

File: README.md [hunk 2]
```diff
@@ -1 +1,119 @@
...
+1. Browse our [Trello board](https://trello.com/b/CRiwsjfr/shanti-ashram-web-development) to find an issue you'd like to work on. If you don't find an existing issue, feel free to create a new one.
+
+2. Fork the repository by clicking the "Fork" button on the top right of this page.
+
+3. Clone your forked repository to your local machine following the steps mentioned in the "Cloning the Repository" section.
+
+4. Create a new branch for your work:
+
+   ```bash
+   git checkout -b fix-issue-XXX
```
[the remark this answers is not in the document]
Comment (timalsinab) [reply README.md]: I changed it to saw_ticket# - Short_Description

[the remark this answers is not in the document]
Comment (timalsinab) [reply README.md]: image
change the # to the ticket number

**Comment by timalsinab** [comment]
@AnishKharel Can you review it?

File: README.md [hunk 3]
```diff
@@ -28,7 +28,6 @@ Our website includes a range of features designed to provide you with comprehens
 - **Objectives:**
 A website that serves as the front-facing information and involvement page for anyone wanting to know more about us and our goals. It will have many features such as a mission statement, timelines, ways to get involved, contact information, and donations.
 
-- **Love:** Building our foundation, we start everything with love. From peTeaching children to love themselves, and those whom they are surrounded with, will give them a lifetime of lessons- especially to help each other in need. Honoring a natural lifestyle, we will teach children to grow their own food, living in an organic lifestyle, which involves a language of love that is able to be passed down from generations to generations. Everything starts with a vision, and this one is an impeccable one; touching lives of many. We are starting from the ground up, and you can help us take this vision- make it a physical reality.
```
Comment (AnishKharel) [inline README.md:33]: Get rid of the structure charity and add the donation link instead of the description (to the GoFundMe).

**Review by AnishKharel — APPROVED** [review]

**Pull request merged**